frontend/src/app/app-routing.module.ts

```diff
@@ -208,21 +203,27 @@ const routes: Routes = [
                       {
                         path: 'git-model',
                         data: {
-                          breadcrumb: 'git models',
+                          breadcrumb: 'Model Sources',
                           redirect: (data: Data) =>
-                            `/project/${data.project?.slug}/model/${data.model?.slug}`,
+                            `/project/${data.project?.slug}/model/${data.model?.slug}/modelsources`,
                         },
                         children: [
                           {
                             path: 'create',
-                            data: { breadcrumb: 'create' },
+                            data: {
+                              breadcrumb: 'Create Git Model',
+                              redirect: (data: Data) =>
+                                `/project/${data.project?.slug}/model/${data.model?.slug}/git-model/create`,
+                            },
                             component: ManageGitModelComponent,
                           },
                           {
                             path: ':git-model',
                             data: {
                               breadcrumb: (data: Data) =>
                                 data.gitModel?.id || '...',
```

Comment:
When clicking the `...`, it redirects to a URL containing undefined, which leads to an error.

Reply:
Good catch. I changed the redirect to "undefined" at all three occurrences of "..." in case the id is not yet given.